keras_cv/models/object_detection/retina_net/retina_net.py

```diff
@@ -116,7 +116,6 @@ def __init__(
         classification_head=None,
         box_head=None,
         evaluate_train_time_metrics=False,
-        name="RetinaNet",
```

Comment:
why removing the name?

Reply:
Added back - wasn't sure if we wanted/needed default names.  No strog opinion